libraries/core/build.gradle.kts

```diff
@@ -52,6 +50,7 @@ dependencies {
     api(libs.androidx.appcompat)
     api(libs.kotlin.coroutines.android)
 
+    debugImplementation(libs.androidx.tracing) // Espresso fails without this
```

Comment:
It seems that one of the libraries uses tracing 1.0 instead of 1.1 which causes Espresso to fail with `No static method forceEnableAppTracing()V in class Landroidx/tracing/Trace`